monasca/templates/grafana-deployment.yaml

```diff
@@ -30,16 +30,29 @@ spec:
           - name: grafana-config
             mountPath: /etc/grafana
           env:
+            - name: GF_AUTH_KEYSTONE_AUTH_URL
+            {{- if .Values.grafana.keystone.url }}
+{{- include "monasca_secret_env" .Values.grafana.keystone.url | indent 14 }}
+            {{- else }}
+              value: "http://{{ template "keystone.fullname" . }}:{{ .Values.keystone.service.admin_port }}"
+            {{- end }}
+{{ include "monasca_keystone_env" .Values.grafana.keystone | indent 12 }}
+            - name: GF_DATABASE_HOST
```

Comment:
so weird two mysql parameters call to the different tpl files

Reply:
Also, I need to set the default host to .Release.name and you can't do that in values.yaml and since I do it so many different places, it made sense to do it in a template